### PR TITLE
Init.properties refactoring + chrome driver update

### PR DESCRIPTION
**Pull request by Atri0**
Two commits
1. Refactoring of Init.properties file, which changes DRILL_USER_NAME and DRILL_USER_PASSWORD fields to ADMIN_USER1_NAME / ADMIN_USER1_PASSWORD fields, also adds one user as admin and two non admin users
2. Chrome driver update

### CODE REVIEW

File: src/test/java/org/apache/drillui/test/framework/testng/unsecure/rest/BaseRestTest.java [hunk 1]
```diff
@@ -23,7 +23,13 @@
 public class BaseRestTest {
   @BeforeSuite
   public void setupRestAssured() {
-    RestAssured.port = Integer.parseInt(TestProperties.get("DRILL_PORT"));
```
Comment (denysord88) [inline src/test/java/org/apache/drillui/test/framework/testng/unsecure/rest/BaseRestTest.java]: @Atri0 Could you please revert the changes to this file? We should have host and port data in the init.properties file.

**Comment by denysord88** [comment]
+1
Please squash the commits before the merge.

**Comment by Atri0** [comment]
@denysord88  Commits were squashed
